### ISSUE TITLE
Compilation flag inference in as-is imports is inconsistent with update behavior

### ISSUE THREAD

**Issue by busybit** [beetbox/beets#1099]
When I import a compilation with "Use as is", and after that run "beet update", the comp: flag is set to false and the album is split by artist into many artist folders.

**Comment by lfam** [comment]
> I'd be interested to hear from any other users about what they expect from "as-is" imports and the update command.

I just experienced this issue and it absolutely not what I expected to happen. As-is imports should stay as-are.

**Comment by DirkReiners** [comment]
This just bit me... The current behavior is really non-intuitive and potentially very destructive (I now have hundreds of directories with one song each...).

My workaround was to manually set the compilation flag (using mid3v2 --TCMP 1 */*mp3) and update again. But I was lucky, the disassembled compilations ended up in one exclusive folder, making that possible. A preventative could be to run mid3v2 --TCMP 1 */Compilations/*/*mp3  */Compilations/*/*/*mp3 before running update, but that's pretty brutal, and you need to know about it before you do the update...

Is anybody working on this?

**Comment by sampsyo** [comment]
No, I don't think anyone's working on it—any interest in picking it up?

So I can understand a little more, what do you think the right fix would be? We could, as two examples, just stop doing comp flag inference at all on as-is imports, or we could change the `update` command instead to do the same thing as `import -A`.

**Comment by m42i** [comment]
As suggested in #2181  I'd refrain from guessing the album artist on importing.
